### DISCUSSION TITLE
Improving Eslint rules

### DISCUSSION THREAD

**Discussion by khenriksson** [hackjunction/hackplatform#447] [General]
Currently it's rather annoying when developing and everything turns red because eslint isn't completely configured correctly, as we can see from the terminal as well during development. I would suggest going through the eslint config and check out the way we want to configure the project, and maybe also add prettier on the side.

**Comment by pholm** [comment]
I noticed the same while uprading linting related libraries. (https://github.com/hackjunction/JunctionApp/pull/453) Especially in the new version where build fails if there is lint errors, this would be quite essential.
